test/jit/test_parametrization.py

```diff
@@ -43,27 +43,3 @@ def test_traceable(self):
                                     'Cannot trace a model while caching'):
             with parametrize.cached():
                 traced_model = torch.jit.trace_module(model, {'forward': x})
-
-    def test_scriptable(self):
```

Comment:
Hmm, why are you deleting this test?

Reply:
As I mentioned in the PR description, the test was bad and had a TODO to actually implement and just tested that UnsupportedNodeError is thrown, and with GeneratorExp support a different error would be thrown.

Reply:
This test was here on purpose to make sure that trying to script this will raise a nice error message and not crash in an obscure way. I think we want to keep it and make sure the new error message is good.

Reply:
@albanD any suggestions how to best reconcile this? It was just an accident that GeneratorExp was not supported and the test produced somewhat related-looking UnsupportedNodeError.

Reply:
> In that case, I guess we can add a check if it it scripting and if so, raise a nice error?

Hmm, add a check where exactly?

Reply:
@albanD I've updated the PR, please take another look.